charts/gardener/admission-local/charts/runtime/values.yaml

```diff
@@ -19,3 +19,10 @@ webhookConfig:
 service:
   topologyAwareRouting:
     enabled: false
+livenessProbe:
+  enable: true
+readinessProbe:
+  enable: true
+leaderElection:
+  enable: true
+highAvailability: true
```

Comment:
Minor detail: Should the `highAvailability` option follow the same structure as the other options?
```yaml
  highAvailability:
    enable: true
```